src/xss-filters.js

```diff
@@ -80,6 +82,29 @@ exports._getPrivFilters = function () {
             return String.fromCharCode((codePoint >> 10) + 0xD800, (codePoint % 0x400) + 0xDC00);
         };
 
+    // patch document.write() and document.writeln() to properly handle NULL for IE 9 or below
```

Comment:
we never know the exact implementation of document.write in different browsers, it does not look good to override any DOM object and its api.